pkg/networkservice/connectioncontext/dnscontext/client.go

```diff
@@ -169,3 +165,45 @@ func (c *dnsContextClient) initialize() {
 
 	c.updateCorefileQueue.AsyncExec(c.updateCorefile)
 }
+
+func removeDuplicates(elements []*networkservice.DNSConfig) []*networkservice.DNSConfig {
+	var result []*networkservice.DNSConfig
+
+	for _, e := range elements {
+		exists := false
+		for _, v := range result {
+			if equal(e.DnsServerIps, v.DnsServerIps) && equal(e.SearchDomains, v.SearchDomains) {
+				exists = true
+				break
+			}
+		}
+		if !exists {
+			result = append(result, e)
+		}
+	}
+
+	return result
+}
```

Comment:
What if we have two dns configs with different len but have that have intersection? Please handle correctly corner cases.